Form-Controls/styles.css

```diff
@@ -0,0 +1,74 @@
+
+body{
+    background: #F68E5F;  
+    text-align: center;
+    width: 30%;
+    margin:auto;
+}
+main{
+    display: flex;
+    flex-direction: column;
+    align-items:center  ;
+    justify-content: center;
+    background: #F5DD90;
+    border-radius: 20px;
+    padding-bottom: 20px;
+}
+
+#heading{
```

Comment:
Try to avoid the use of #ids as much as possible in CSS. IDs are extremely powerful; ids outweigh an infinite number of class and element and attribute selectors, so they can be hard to override. Here's a good article digging in to this a bit more : https://css-tricks.com/almanac/selectors/i/id/